src/assets/stylesheets/components/_today-date.scss

```
@@ -0,0 +1,18 @@
+.today-date {
+  &__date {
+    font-size: map-get($font-sizes, xs);
+    font-weight: $font-weight-bold;
+
+    letter-spacing: rem(-0.08px);
```

```suggestion
    letter-spacing: -0.08px;
```
It is such a small amount, should we leave it with out `rem`?